README.md

```diff
@@ -10,13 +10,13 @@ but has since grown up to work on all the platforms (Xamarin.*, .NET, UWP, Azure
 SQLite-net was designed as a quick and convenient database layer. Its design follows from these *goals*:
 
 * Very easy to integrate with existing projects and runs on all the .NET platforms.
-  
```

Comment:
I have Sublime Text set to automatically remove unneeded whitespace at the end of lines, which is why this PR also includes the removal of these unneeded whitespaces.